suzieq/poller/worker/services/bgp.py

```diff
@@ -96,10 +96,7 @@ def _clean_eos_data(self, processed_data, raw_data):
                         new_entry['communityTypes'] = ['standard', 'extended']
                 else:
                     new_entry['communityTypes'] = ['standard']
-                if entry.get('hopsMax', '').isnumeric():
-                    # Older versions of EOS don't provide this field
-                    new_entry['hopsMax'] = int(entry.get('hopsMax', 255))-255
```

Comment:
We don't need to do this anymore, right?

Reply:
Correct, this was the only bug that needed to be fixed for capturing the TTL info correctly for multihop eBGP